azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/internal/BaseAzureComputeApiLiveTest.java

```diff
@@ -99,6 +100,7 @@ public void setup() {
       vaultResourceGroup = System.getProperty("test.azurecompute-arm.vault.resource.group");
       vaultName = System.getProperty("test.azurecompute-arm.vault.name");
       vaultCertificateUrl = System.getProperty("test.azurecompute-arm.vault.certificate.url");
+      tenantId = System.getProperty("test.azurecompute-arm.tenantId");
```

Comment:
Change to: `tenantId = injector.getInstance(Key.get(String.class, Tenant.class));`